libs/message-coder/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@votingworks/message-coder",
+  "version": "1.0.0",
+  "description": "Encode and decode messages for binary communication.",
+  "keywords": [],
+  "license": "AGPL-3.0",
+  "author": "VotingWorks Eng <eng@voting.works>",
+  "main": "build/index.js",
+  "scripts": {
+    "build": "tsc --build tsconfig.build.json",
+    "clean": "rm -rf build tsconfig.tsbuildinfo tsconfig.build.tsbuildinfo",
+    "lint": "pnpm type-check && eslint .",
+    "lint:fix": "pnpm type-check && eslint . --fix",
+    "pre-commit": "lint-staged",
+    "test": "is-ci test:ci test:watch",
```

Comment:
Unrelated to the content of this PR, I have been confused about this series of commands. `is-ci test:ci test:watch`. It seems like three different commands slammed together, how is this syntax working?

Reply:
[`is-ci` takes the names of `npm` scripts as the first two arguments](https://github.com/YellowKirby/is-ci-cli/blob/6a11a14bec65dcbbb6ff52fdb468827157ee1be3/cli.js#L25), then calls `npm run …` with the appropriate one.